core/templates/pages/learner-group-pages/edit-group/learner-group-preferences.component.spec.ts

```diff
@@ -466,4 +452,26 @@ describe('LearnerGroupPreferencesComponent', () => {
       expect(component.invitedLearnersInfo).toEqual([]);
     })
   );
+
+  it('should successfully delete learner group', fakeAsync(() => {
+    spyOn(ngbModal, 'open').and.returnValue({
+      componentInstance: {
+        learnerGroupTitle: learnerGroup.title,
+      },
+      result: Promise.resolve()
+    } as NgbModalRef);
+    spyOn(learnerGroupBackendApiService, 'deleteLearnerGroupAsync')
+      .and.returnValue(Promise.resolve(true));
+    spyOn(loaderService, 'showLoadingScreen');
+
+    component.learnerGroup = learnerGroup;
+
+    component.deleteLearnerGroup();
+    tick(100);
```

Comment:
Please add a comment explaining where this number was obtained from.

Reply:
It was not needed, removed this. Thanks!